bin/setup

```diff
@@ -23,12 +23,24 @@ chdir APP_ROOT do
   #   cp 'config/database.yml.sample', 'config/database.yml'
   # end
 
-  puts "\n== Preparing database =="
-  system! 'bin/rails db:setup'
+  puts "\n== Running bundler =="
+  system! 'bundle install'
 
-  puts "\n== Removing old logs and tempfiles =="
-  system! 'bin/rails log:clear tmp:clear'
+  puts "\n== Setting Workspace =="
+  system! 'bin/rake setup_workspace'
 
-  puts "\n== Restarting application server =="
```

Comment:
What was this removed?

Reply:
Because it was moved to a npm script: It makes easier to automate the setup process on npm script. Same principle: Cleaning the automation script.